compose.yml

```
@@ -0,0 +1,46 @@
+services:
+
+  bootstrap:
+    build:
+      context: .
+      dockerfile: Dockerfile.libp2p
+    ports:
+      - "3000:3000"
+      - "8080:8080"
```

Indicate clearly if it's TCP or UDP:
```suggestion
      - "3000:3000/tcp"
      - "8080:8080/tcp"
```